update for zero padded ids, add screenshots

lsteck opened the pull request:
Updated workshop scripts to zero pad the user ids so update documentation to correspond with changes.  
planning/817

Also Added screen shots to these labs to help users follow along with the stpes.

csantanapr commented on src/pages/workshop/appmod/index.mdx:
````diff
@@ -36,21 +36,21 @@ import Globals from 'gatsby-theme-carbon/src/templates/Globals';
+1. Set `TOOLKIT_USERNAME` environment variable, replace `userXX` with assigned username (ex. `user01`)
     ```bash
-    TOOLKIT_USERNAME=user1
+    TOOLKIT_USERNAME=userXX
````
I think we want this to be `user01` this way user can just copy and paster without modification if they are using the openlabs where he/she is the only user.

lsteck commented:
@csantanapr I updated the pages as suggested.  Added a `userdemo` and updated the scripts as well to create this user.

Also I updated the setup, I split out the terminal configuration for the workshop participants and I documented how to customize the workshop setup.

Thanks!

csantanapr reviewed (approved):
LGTM
